waku/node/v2/wakunode2.nim

```diff
@@ -47,6 +47,30 @@ proc initAddress(T: type MultiAddress, str: string): T =
     raise newException(ValueError,
                        "Invalid bootstrap node multi-address")
 
+proc removeContentFilters(filters: var Filters, contentFilters: seq[ContentFilter]) {.gcsafe.} =
```

Comment:
Maybe some trace/debug logs here? To see updated content filter state

Reply:
Added in cb146e3